edflow/iterators/template_iterator.py

```diff
@@ -227,3 +228,34 @@ def step_op(self, model, **kwargs):
         written to the log file and stdout. Outputs of 'eval_op' are written
         into the project's eval folder to be evaluated with `edeval`."""
         raise NotImplemented()
+
+    def get_fixed_examples(self, names):
+        """collect fixed examples from dataset with names.
+        
+        Examples
+        --------
+
+        If the dataset returns the examples ["image", "stickman"], (as in VUnet),
+        `self.get_fixed_examples(["image", "stickman"])` will return
+        {
+            "image": batch of self.dset[i]["image"] with fixed i,
+            "stickman": batch of self.dset[i]["stickman"] with fixed i
+        }
+
+        TODO: document usage somewhere
```

Comment:
my best guess: this will never happen once that todo is committed. I suggest doing the todos immediately or removing the todo comments

Reply:
Done